libs/components/forms/src/lib/modules/character-counter/character-counter.directive.ts

```diff
@@ -70,24 +72,26 @@ export class SkyCharacterCounterInputDirective implements Validator {
         },
       };
     }
+
+    return {};
   }
 
   public registerOnValidatorChange(fn: () => void): void {
-    this._validatorChange = fn;
+    this.#_validatorChange = fn;
   }
 
-  private updateIndicatorCount(count: number): void {
+  #updateIndicatorCount(count: number): void {
     if (this.skyCharacterCounterIndicator) {
       this.skyCharacterCounterIndicator.characterCount = count;
     }
   }
 
-  private updateIndicatorLimit(limit: number): void {
+  #updateIndicatorLimit(limit: number): void {
     if (this.skyCharacterCounterIndicator) {
       this.skyCharacterCounterIndicator.characterCountLimit = limit;
     }
   }
 
   // eslint-disable-next-line @typescript-eslint/no-empty-function
-  private _validatorChange = () => {};
+  #_validatorChange = () => {};
```

Comment:
Should probably just be `#validatorChange` since it's not a getter/setter.

Reply:
Good catch. Done